conda-store-server/tests/conftest.py

```diff
@@ -290,6 +290,20 @@ def plugin_manager():
     return pm
 
 
+@pytest.fixture
+def alembic_config(conda_store):
+    from conda_store_server._internal.dbutil import write_alembic_ini
+
+    ini_file = pathlib.Path(__file__).parent / "alembic.ini"
+    write_alembic_ini(ini_file, conda_store.database_url)
+    return {"file": ini_file}
+
+
+@pytest.fixture
+def alembic_engine(db):
+    return db
```

Comment:
I see the `pytest-alembic` docs mention something about defining an `alembic_engine`:

> Override this fixture to provide pytest-alembic powered tests with a database handle.

But this function doesn't seem to do anything. Do the tests run without this? If not, it would be helpful to have a docstring here about it being necessary to avoid future maintainer confusion.

Reply:
good eye, tests continue to work without this. Will remove it :x: